src/engine/QueryPlanner.cpp

```diff
@@ -875,6 +876,23 @@ vector<QueryPlanner::SubtreePlan> QueryPlanner::seedWithScansAndText(
           "necessary also rebuild the index.");
     }
 
+    const string keyword = "MaxDistanceInMeter:";
```

Comment:
```suggestion
    const string keyword = "MaxDistanceInMeters:";
```
And add this magic constant to the `global/Constants.h` file and add some comments there as for what the triples have to look like.

Reply:
And maybe use the prefix for the internal URIs (see the constants file there, otherwise we can discuss this next time in person).